src/assets/scss/_my-account-personal-details.scss

```diff
@@ -44,6 +44,9 @@
   .active {
     background: $wild-sand;
   }
+  button {
+    text-transform: uppercase;
+  }
```

Comment:
Could you put it in the component? As this is a temporary solution, we won't forget to remove it if it's in the component.